infrastructure/create-github-environment.js

```diff
@@ -38,21 +50,27 @@ const config = {
     GATEWAY_HOST: ''
   },
   smtp: {
-    SMTP_HOST: process.env.SMTP_HOST || '',
-    SMTP_USERNAME: process.env.SMTP_USERNAME || '',
-    SMTP_PASSWORD: process.env.SMTP_PASSWORD || '',
-    EMAIL_API_KEY: process.env.EMAIL_API_KEY || '',
+    SMTP_HOST: process.env.SMTP_HOST,
+    SMTP_USERNAME: process.env.SMTP_USERNAME,
+    SMTP_PASSWORD: process.env.SMTP_PASSWORD,
     SMTP_PORT: '',
     ALERT_EMAIL: ''
```

Comment:
I missed this. This should be the mailbox associated with the Slack redirect and is a requirement